cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -100,6 +100,13 @@ jobs:
           name: artifacts
           path: dist-manifest.json
 
+{{%- for job in plan_jobs %}}
+
+  custom-{{{ job|safe }}}:
+    uses: ./.github/workflows/{{{ job|safe }}}.yml
+    secrets: inherit
+{{%- endfor %}}
+
   # Build and packages all the platform-specific things
   build-local-artifacts:
     name: build-local-artifacts (${{ join(matrix.targets, ', ') }})
```

Comment:
Alright so welcome to the real hell:

every single task that depends on a builtin job needs to be changed to depend on the builtin job *and all custom jobs in that class*.

Reply:
So, do we actually? Or can they simply be a second stream of jobs that continues parallel?